packages/apps/helpdesk/src/screens/TicketFormScreen.js

```diff
@@ -16,267 +16,100 @@
-          />
-          <DateInput
-            onDateChange={value => handleTicketFieldChange(value, 'startDateT')}
-            title={I18n.t('Helpdesk_StartDate')}
-            defaultDate={
-              _ticket?.startDateT != null ? new Date(_ticket?.startDateT) : null
-            }
-            style={styles.input}
-            nullable={true}
-          />
-          <DateInput
-            onDateChange={value => handleTicketFieldChange(value, 'endDateT')}
-            title={I18n.t('Helpdesk_EndDate')}
-            defaultDate={
-              _ticket?.endDateT != null ? new Date(_ticket?.endDateT) : null
-            }
-            style={styles.input}
-            nullable={true}
-          />
-          <DateInput
-            onDateChange={value =>
-              handleTicketFieldChange(value, 'deadlineDateT')
-            }
-            title={I18n.t('Helpdesk_DeadlineDate')}
-            defaultDate={
-              _ticket?.deadlineDateT != null
-                ? new Date(_ticket?.deadlineDateT)
-                : null
-            }
-            style={styles.input}
-            nullable={true}
-          />
-          <DurationFormInput
-            duration={_ticket?.duration}
-            onChange={value => handleTicketFieldChange(value, 'duration')}
-          />
-          <UserSearchBar
-            titleKey="Helpdesk_AssignedToUser"
-            placeholderKey="Helpdesk_AssignedToUser"
-            defaultValue={_ticket?.assignedToUser}
-            onChange={value => handleTicketFieldChange(value, 'assignedToUser')}
-            style={styles.picker}
-            styleTxt={styles.marginTitle}
-          />
-          <UserSearchBar
-            titleKey="Helpdesk_ResponsibleUser"
-            placeholderKey="Helpdesk_ResponsibleUser"
-            defaultValue={_ticket?.responsibleUser}
-            onChange={value =>
-              handleTicketFieldChange(value, 'responsibleUser')
-            }
-            style={styles.picker}
-            styleTxt={styles.marginTitle}
-          />
-          <FormHtmlInput
-            title={I18n.t('Base_Description')}
-            onChange={value => handleTicketFieldChange(value, 'description')}
-            defaultValue={_ticket?.description}
-          />
-        </View>
-      </KeyboardAvoidingScrollView>
-      <View style={styles.button_container}>
-        <Button
-          title={I18n.t('Base_Save')}
-          onPress={idTicket != null ? updateTicketAPI : createTicketAPI}
-          disabled={disabledButton}
-          color={disabledButton ? Colors.secondaryColor : Colors.primaryColor}
-        />
-      </View>
-    </Screen>
+    <FormView
+      formKey="helpdesk_ticket"
+      defaultValue={defaultValue}
+      actions={[
+        {
+          key: 'create-ticket',
+          type: 'create',
+          needValidation: true,
+          needRequiredFields: true,
+          hideIf: () => idTicket != null,
+          customAction: ({dispatch, objectState}) => {
```

Comment:
you can use "handleReset" which is given in argument to reset form view to default value